Dockerfile

```diff
@@ -8,6 +8,14 @@ ARG TAG=pinned
 
 # Build Postgres
 FROM $REPOSITORY/$IMAGE:$TAG AS pg-build
+
+# Install and update packages.
+USER root
+RUN apt-get update &&       \
+    apt-get install -y curl \
+                       jq   \
+                       netcat \
+                       valgrind
```

Comment:
This should go to the apt-get in the final image below

Reply:
Done.